presto-main/src/test/java/com/facebook/presto/sql/query/TestLegacyUnnestArrayRows.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.query;
+
+import com.facebook.presto.sql.analyzer.FeaturesConfig;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+
+public class TestLegacyUnnestArrayRows
+{
+    @Test
+    public void testLegacyUnnestArrayRows()
+    {
+        QueryAssertions assertions = new QueryAssertions(testSessionBuilder().build(), new FeaturesConfig().setLegacyUnnestArrayRows(true));
```

Comment:
This needs to be closed:

```java
try (QueryAssertions queryAssertions = new QueryAssertions(...)) {
```

Reply:
Also, you could just do this instead of adding FeaturesConfig to the QueryAssertions:

```
testSessionBuilder()
    .setSystemProperty(LEGACY_UNNEST, "true")
    .build()
```

Although, admittedly, that doesn't test the whole config->session->engine path.